src/win/win_config_manager.h

```diff
@@ -3,13 +3,22 @@
 
 #include <string>
 #include "i_config_manager.h"
+#include "win_path_manager.h"

```

Comment:
Move `includes` needed only in the source file to the source file. Otherwise including this header will always include these libs/files.